source/_config.yml

```
@@ -8,7 +8,7 @@ copyright : "2020–2022"
 # Force re-execution of notebooks on each build.
 # See https://jupyterbook.org/content/execute.html
 execute:
-  execute_notebooks: auto
+  execute_notebooks: force
```

Revert the changes before merging.
```suggestion
  execute_notebooks: auto
```